File: solutions/kotlin/sum-of-multiples/1/src/main/kotlin/SumOfMultiples.kt
object SumOfMultiples {

    fun sum(factors: Set<Int>, limit: Int): Int {
        var ans = 0
        for (i in 1 until 20) {
            var flag = true
            for (f in factors) {
                if (i % f == 0) {
                    flag = false
                    break
                }
            }
            if (flag) {
                ans += i
            }
        }
        return ans
    }
}

File: solutions/julia/sum-of-multiples/1/sum-of-multiples.jl
function sum_of_multiples(limit, factors)
    b = [false for _ in 1:(limit-1)]
    for f in factors
        g = f
        while f != 0 && g < limit
            b[g] = true
            g += f
        end
    end
    sum([i for i in 1:(limit-1) if b[i]])
end
